client/components/singleentry/DetailTable.js

```diff
@@ -2,10 +2,63 @@ import React from 'react'
 
 export default class DetailTable extends React.Component {
 
+  constructor (props) {
+    super(props)
+
+    var entries = []
+
+    for (var item in props.myData) {
+      if (item !== 'tags') {
+        entries.push({
+          name: item,
+          value: props.myData[item]
+        })
+      }
+    }
+
+    if (props.myData['tags']) {
+      var i = 0

```

Comment:
can you use better naming instead of "i"? Could reveal much more hints. Also you can use `let` instead of var, if it's inside the scope and not global.